src/EditorFeatures/TestUtilities/Classification/AbstractClassifierTests.cs

```diff
@@ -46,9 +46,9 @@ public abstract class AbstractClassifierTests
             var span = new TextSpan(start, length);
             var actual = await GetClassificationSpansAsync(allCode, span, parseOptions, testHost);
 
-            actual = actual.Sort((t1, t2) => t1.TextSpan.Start - t2.TextSpan.Start);
+            var actualOrdered = actual.OrderBy((t1, t2) => t1.TextSpan.Start - t2.TextSpan.Start);
```

Comment:
Add a comment this is a stable sort intentionally?

Reply:
i thought about the same.  but as a test helper, i don't care :)